site/website/blog/2020-07-16-designing-evergreen-cache-cdc.md

```diff
@@ -205,16 +205,14 @@ data, depending on the underlying disk technology.
 
 [Hazelcast Jet](https://jet-start.sh/) is a distributed stream
 processing framework built on Hazelcast and combines a cache with
-fault-tolerant data processing.
-It has sources and sinks to integrate with several file, messaging and
-database systems (such as Amazon S3, Kafka, message brokers and
-relational databases).
+fault-tolerant data processing. It has sources and sinks to integrate
+with various file-, messaging- and database systems (such as Amazon S3,
+Kafka, message brokers and relational databases).
 
 Jet also provides a Debezium module where it can process change events
 directly from the database and write them to its distributed key-value
-store.
-This avoids having to write the intermediate messages to Kafka and then
-read again to be written to a separate cache.
+store. This avoids having to write the intermediate messages to Kafka
```

Comment:
Can you keep the line breaks? It renders the same (or so I hope), and is easier on the eyes for reading.

Reply:
We don't use arbitrary line breaks, if it's the same paragraph, it should flow until the margin.

Reply:
I can't really decide if it's easier on my eyes or if it annoys me greatly. I think I will just not touch it again.

Reply:
IIRC there were many instances of this in the document and it annoyed me: it looks like a new paragraph, but actually it's not.

Reply:
Those LB are **not** arbitrary. I put them after each ., : or ;. It helps a lot while writing/reading a post.

I don't see the point in removing them in the course of a PR that has nothing to do with it.
It's similar to changing tabs to spaces while fixing a bug.
But since you all agree, it's up to you now.

Reply:
I'm with Nicolas on this (can't believe I'm actually uttering these words... ;) ). I don't particularly like his formatting style either, but it's not harming anybody, so we shouldn't mess with it, especially not in a completely unrelated PR. I will revert my initial "fixes".

Reply:
I didn't realize this was a blog post, we can have some personal style in them, but for the docs that are team-maintained, we have some policies on formatting.